README.md

```diff
@@ -15,34 +15,47 @@
 
 ## How to use
 
-1. **Install the 'JavaScript Standard Style' extension**
+1. **Install the 'StandardJS - JavaScript Standard Style' extension**
 
-   If you don't know how to install extensions in VSCode, take a look at the [documentation](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-and-install-extensions).
+    Launch VSCode Quick Open (⌘+P), paste the following command, and press enter.
```

Comment:
What about making easier for the user to install the extension instead of redirecting them to the documentation ?

We're still redirecting them to the documentation if they want more information.